### PR TITLE
2.1.1

### PR DESCRIPTION
**Pull request by holl-**
This is a bug fix release.

* Add to `Solve.preprocess_y_args` to prevent jit issues
* Fix display of large curves in Dash
* Add workaround for [PyTorch jit double execution](https://github.com/pytorch/pytorch/issues/75655)
* Fix `dense_net` implementations, support batch normalization
* Documentation updates

### CODE REVIEW

**Review by thunil — APPROVED** [review]
My tests run through so far
